api/types/events/events.proto

```diff
@@ -1319,6 +1333,7 @@ message OneOf {
         events.BillingInformationUpdate BillingInformationUpdate = 46;
         events.BillingCardCreate BillingCardCreate = 47;
         events.BillingCardDelete BillingCardDelete = 48;
+        events.AccessRequestDelete AccessRequestDelete = 66;
```

Comment:
The 66 is intentional here, for compatibility with newer versions, correct?

Reply:
Yes, it matches the field number for `AccessRequestDelete` in `master`; there's a bunch of event types that haven't been backported, especially not to v6.2